resource-definitions/k8s-cluster-aks/agent/aks-agent.tf

```diff
@@ -0,0 +1,24 @@
+# AKS private cluster. It is to be accessed via the Humanitec Agent
+# It is using a Cloud Account to obtain credentials
+resource "humanitec_resource_definition" "aks-agent" {
+  id          = "aks-agent"
+  name        = "aks-agent"
+  type        = "k8s-cluster"
+  driver_type = "humanitec/k8s-cluster-aks"
+  # The driver_account is referring to a Cloud Account resource
+  driver_account = humanitec_resource_account.azure-dynamic.id
+
+  driver_inputs = {
+    values_string = jsonencode({
+      "name"            = var.azure_aks_private_cluster_name
+      "resource_group"  = var.azure_aks_resource_group
+      "subscription_id" = var.azure_subscription_id
```

Comment:
Don't we need `loadbalancer` here as well?

Reply:
It's not mandatory, but added it anyway to be consistent with the YAML example, much better. Thanks!